main/systemd.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "strings"
+    "path/filepath"
+)
+
+func contains(distros []string, distro string) bool {
+    if distro != "" {
+        for _, v := range distros {
+            if v == distro {
+                return true
+            }
+        }
+    }
+    return false
+}
+
+func get_distro() string {
```

Comment:
NIT :Function names in golang follow `camelCasing` over `snake_casing`. Some normal conventions - https://www.golangprograms.com/naming-conventions-for-golang-functions.html